internal/cobraext/const.go

```diff
@@ -20,6 +20,9 @@ const (
 	VerboseFlagName        = "verbose"
 	VerboseFlagDescription = "verbose mode"
 
+	StackServicesFlagName        = "services"
+	StackServicesFlagDescription = "component services (comma-separated values: elasticsearch, kibana, package-registry)"
```

Comment:
How do you feel about introducing a `%s` placeholder instead of hardcoding the service names here and then interpolating it with `availableServices` where this constant is used?

Reply:
Fixed